xwiki-platform-core/xwiki-platform-search/xwiki-platform-search-ui/src/main/resources/XWiki/QuickSearchUIX.xml

```diff
@@ -241,7 +241,8 @@
   #if ($displayQuickSearch &amp;&amp; ($xwiki.exists('Main.Search') || $xwiki.exists('Main.WebSearch')))
     #set ($discard = $xwiki.jsx.use('XWiki.QuickSearchUIX'))
     &lt;li&gt;
-      &lt;form class="navbar-form globalsearch globalsearch-close form-inline" id="globalsearch" action="#if($xwiki.exists('Main.Search'))$xwiki.getURL('Main.Search')#else$xwiki.getURL('Main.WebSearch')#end"&gt;
+      &lt;form class="navbar-form globalsearch globalsearch-close form-inline" id="globalsearch" 
+        action="#if($xwiki.exists('Main.Search'))$xwiki.getURL('Main.Search')#else$xwiki.getURL('Main.WebSearch')#end" role="search"&gt;
```

Comment:
This line is too long (the search URL could be computed before) and it should be indented to make it clear that it's different than the next line with the child label element.

Reply:
Is this the right approach to compute search URL?
`#set  ($searchUrl = 
    #if
 ($xwiki.exists('Main.Search'))$xwiki.getURL('Main.Search')#else$xwiki.getURL('Main.WebSearc')
    #end
) `

and then 

` action="$searchUrl" `


I will make the indentations perfect after it